flask/README.md

```diff
@@ -64,19 +64,20 @@ Since we are working with docker you do not have to install a local MySQL server
 
 #### General notes on Docker network
 
-In the docker-compose file we define a separate docker network called `backend` to which the back-end containers are joined. Each container can now look up the hostname `flask` or `mysql` and get back the appropriate container’s IP address.
-To access the mysql database, there are now two possibilities:
+In the docker-compose file we define a separate docker network called `backend` to which the back-end containers are joined. Each container can now look up the host name `flask` or `mysql` and get back the appropriate container’s IP address.
+To access the mysql database, there are now three possibilities:
 
 1. You reach the mysql db at `MYSQL_HOST=mysql` and `MYSQL_PORT=3306` or
-2. by specifying the IP-address of the gateway for `MYSQL_HOST` and `MYSQL_PORT=32000`.
+1. You execute the mysql command line client in the running container by `docker-compose exec mysql mysql -u root -p` or
```

Comment:
Use the correct numbers please @pylipp  (1,2,3)

Reply:
It's a markdown feature that this an enumeration can be indicated by `1., 1., 1.` and will be rendered as `1., 2., 3.`. I find it helpful because I don't have to change surrounding numbers when I insert an enumeration point inbetween. Should I still update the numbers?

Reply:
Ah, I didn't know that trick. Good for me to know too ;). Let's keep as is